subscriptions/fixed-price/checkout/node/src/server.ts

```diff
@@ -207,3 +247,248 @@ app.post('/webhook', async (req, res) => {
 })
 
 app.listen(3000, () => console.log('Running on port 3000'))
+
+/***************************************** Handlers ***************************************************/
+
+/**
+ * Handler should be called when you are asked to sign a charge request.
+ * @param diagonal Diagonal SDK instance
+ * @param signatureRequest The signature request event data
+ */
+async function handleSignatureChargeRequest(
+  diagonal: Diagonal,
+  signatureRequest: Signature,
+): Promise<void> {
+  // 1: Extract charge payload and optionally verify charge request
+  const charge = signatureRequest.data.charge
+
+  //2: Use your signing key and attest to the charge request
+  const ecdsaSignature = diagonal.signatures.sign(signatureRequest, signingKey)
+
+  // 3: Capture the charge request
+  await diagonal.charges.capture(charge.id, ecdsaSignature)
+}
+
+/**
+ * Charge confirmed
+ *
+ *  We recommend pivoting on `charge.confirmed` when handling the following subscription lifecycle events:
```

Comment:
How about we change the `pivoting` grammar, as it is a bit confusing and requires a higher level understanding of english language?

My recommendation is:

`We recommend listening to `charge.confirmed` events to handle the following subscription lifecycle events:`